recipes/quest/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "quest" %}
+{% set version = "3.0-a1" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
```

Comment:
According to https://conda.io/docs/user-guide/tasks/build-packages/package-spec.html#info-index-json: "The package version. May not contain “-“. Conda acknowledges PEP 440."

Above, your version includes a `-`.

Reply:
@ceball this will get changed. Thanks